api/src/main/java/io/opencensus/trace/EndSpanOptions.java

```diff
@@ -53,16 +50,18 @@ public static Builder builder() {
    * io.opencensus.trace.export.SampledSpanStore#registerSpanNamesForCollection(Collection)} API
    * instead.
    *
-   * @return this.
+   * @return {@code true} if the name of the {@code Span} should be registered to the {@code
+   *     io.opencensus.trace.export.SampledSpanStore}.
    */
   @ExperimentalApi
   public abstract boolean getSampleToLocalSpanStore();
 
   /**
    * Returns the status.
    *
-   * @return the status.
+   * @return the status or {@code null} if not set.
```

Comment:
IIUC this will also return null if the status is explicitly set to null.

Reply:
Better to not allow it to be set to null. So comment is fine.

Reply:
It looks like setStatus never checks for null.  I don't know if it's possible with the AutoValue builder, though.

Reply:
Don't think this is needed.